tests/test_main.py

```diff
@@ -34,3 +33,150 @@ def test_post_song():
+    assert response.status_code == 200
+
+
+def test_post_song2():
+    response_post = post_song2(
+        "post_test_song",
+        "post_test_description",
+        "post_test_creator",
+        "post_test_artists",
+        "./tests/test.song",
+        {"api_key": "key"},
+    )
+    assert response_post.status_code == 200
+    response_get = client.get(
+        "/api/v2/songs/" + str(response_post.json()["id"]),
+        headers={"api_key": "key"},
+    )
+    assert str(response_get.json()["id"]) == str(response_post.json()["id"])
+    assert response_get.json()["name"] == "post_test_song"
+    assert response_get.json()["description"] == "post_test_description"
+    assert response_get.json()["creator"] == "post_test_creator"
+    assert response_get.json()["artists"] == "post_test_artists"
+
+
+def test_put_song2():
+    response_post = post_song2(
+        "put_test_song",
+        "put_test_description",
+        "put_test_creator",
+        "put_test_artists",
+        "./tests/test.song",
+        {"api_key": "key"},
+    )
+    assert response_post.status_code == 200
+
+    response_update = client.put(
+        "/api/v2/songs/" + str(response_post.json()["id"]),
+        data={
+            "name": "updated_test_song",
+            "artists": "updated_test_artists",
+        },
+        headers={"api_key": "key"},
+    )
+    assert response_update.status_code == 200
+    assert response_update.json()["id"] == response_post.json()["id"]
+
+    response_get = client.get(
+        "/api/v2/songs/" + str(response_post.json()["id"]),
+        headers={"api_key": "key"},
+    )
+
+    assert str(response_get.json()["id"]) == str(response_post.json()["id"])
+    assert response_get.json()["name"] == "updated_test_song"
+    assert response_get.json()["description"] == "put_test_description"
+    assert response_get.json()["creator"] == "put_test_creator"
+    assert response_get.json()["artists"] == "updated_test_artists"
+
+
+def test_get_song_by_creator2():
+
+    for i in range(3):
+        response_post = post_song2(
+            "byCreator_test_song" + str(i),
+            "byCreator_test_description",
+            "byCreator_test_creator",
+            "byCreator_test_artists",
+            "./tests/test.song",
+            {"api_key": "key"},
+        )
+        assert response_post.status_code == 200
+    for i in range(3):
+        response_post = post_song2(
+            "notByCreator_test_song" + str(i),
+            "notByCreator_test_description",
+            "notByCreator_test_creator",
+            "notByCreator_test_artists",
+            "./tests/test.song",
+            {"api_key": "key"},
+        )
+        assert response_post.status_code == 200
+
+    response_get = client.get(
+        "/api/v2/songs/?creator=byCreator_test_creator",
+        headers={"api_key": "key"},
+    )
+
+    assert len(response_get.json()) >= 3  # local runs may use a dirty db
```

Comment:
Creo que podemos solucionarlo implementando [esto](https://stackoverflow.com/questions/67255653/how-to-set-up-and-tear-down-a-database-between-tests-in-fastapi)

Reply:
Genial, Suena piola lo de los rollbacks